docs/release-notes/mia-platform-v11.mdx

```diff
@@ -0,0 +1,134 @@
+Therefore, to better enhance this integration, we have introduced a proper governance system, and this feature has precisely this goal: to make it easier to keep constantly documented and ordered all the Providers you want the Console to use. Through the new dedicated section - named "Providers" - inside the "Company Overview" area it will be possible to manage the creation, configuration, and editing of the providers used by Mia-Platform Console (GitProvider, Secret Manager and CI/CD Tools).
+
+![Default Providers Selection](./img/overview_v11/defaultProvider.png)
+
+Furthermore, again through the same section, it will be possible to set the default Providers that will be used in all new projects created in the respective company. This will enhance the governance of Company Owners on all the Projects of their Companies, granting them better control over the management of company tools.
+
+### Machine-to-Machine Authentication
+
+Service accounts are an essential component of many IT environments, as they allow applications and services to access various resources and perform operations without the need for human intervention.
+Assigning roles and permissions to Service accounts lets you improve the speed, reliability, and scalability of your CI/CD workflows.
+Service accounts may be suitable for those applications that require continuous monitoring, reporting, and data processing; alternatively, you can use them also to streamline the process of deploying code changes to an application, for instance by assigning them permissions to trigger automatic deploys.
+Moreover, they can also be great for security concerns: using Service accounts promotes the separation of roles and responsibilities, which reduces the risk of credential misuse or theft.
+With Mia-Platform Console you can now create and manage Service accounts, and start to implement automated processes with machine-to-machine communication.
+
+### Webhooks Management
+
+Each user has different needs, especially depending on their business. We believe it is essential to give each user the ability to manage additional custom flows when certain events occur in the Console.
+
+For this reason, we have introduced the possibility for the user to subscribe to specific events in Console via HTTP webhooks, through our CMS. With this feature it will be possible to configure custom HTTP calls that will be triggered when the respective event takes place in Console
+
+For now, the supported events are:
+
+* Project Creation  
+* Service Creation
+
+### New Cluster connection
+
+Mia-Platform v11 supports all the best and most widely used Runtime Services on the market.
+
+Within the Console, the experience of connecting a Cluster has been completely improved, so that you can choose your configuration from a predefined set of officially supported Runtime Services divided by vendor and have a clearer and more intuitive experience.
```

Comment:
```suggestion
Within the Console, the experience of connecting a Cluster has been significantly improved, so that you can choose your configuration from a predefined set of officially supported Runtime Services divided by vendor and have a clearer and more intuitive experience.
```